frontend/lib/src/components/elements/ChatMessage/styled-components.ts

```diff
@@ -60,6 +60,7 @@ export const StyledAvatarBackground = styled.div(({ theme }) => {
     color: lightTheme ? theme.colors.gray90 : theme.colors.white,
     lineHeight: "1",
     fontSize: theme.fontSizes.md,
+    padding: "1rem",
```

Comment:
nit: why is this additional padding needed?

Reply:
ResizeObserver includes the padding within the width size:
https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect

As a result, we need to add the padding here within the styles otherwise it'll size to avatar size and makes it look weird if you check in past commits.